### PR TITLE
feat: Add cozy-logger for services

### PR DESCRIPTION
**Pull request by Merkur39**
A previous commit replaced cozy-logger by cozy-minilog, but in services you must keep cozy-logger.

### CODE REVIEW

File: src/components/helpers.js [hunk 1]
```diff
@@ -83,6 +84,7 @@ import UnknownIcon from 'cozy-ui/transpiled/react/Icons/Unknow'
 import WalkIcon from 'cozy-ui/transpiled/react/Icons/Walk'
 import WorkIcon from 'cozy-ui/transpiled/react/Icons/Work'
 
+const logService = logger('services/recurringPurposes')
```
Comment (paultranvan) [inline src/components/helpers.js]: That does not sounds like a recurringPurposes service ;)

File: src/components/helpers.js [hunk 2]
```diff
@@ -83,6 +84,7 @@ import UnknownIcon from 'cozy-ui/transpiled/react/Icons/Unknow'
 import WalkIcon from 'cozy-ui/transpiled/react/Icons/Walk'
 import WorkIcon from 'cozy-ui/transpiled/react/Icons/Work'
 
+const logService = logger.namespace('services/recurringPurposes')
```
Comment (paultranvan) [inline src/components/helpers.js:87]: That does not look like a recurringPurpose service ;)

**Review by paultranvan — APPROVED** [review]
Thanks!
I'm not sure why bothering with `logService` rather than `log`, but it looks good

File: src/components/helpers.js (hunk 2 above)
Comment (Merkur39) [reply src/components/helpers.js:87]: En effet, c'est le service `timeseriesWithoutAggregateMigration` qui utilise ce log, j'ai revu aussi l'implem

**Comment by Merkur39** [comment]
> Thanks! I'm not sure why bothering with `logService` rather than `log`, but it looks good

Etant donné que maintenant nous préférons cozy-minilog pour les webApp, et que les deux se côtoie dans cette dernière, il est je pense bien d'être claire :)

**Pull request merged**